sdk/resources/arm-resources/src/lro/locationPolling.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+// Copyright (c) Microsoft Corporation.
```

Comment:
This is weird to have both copyright banners, @sarangan12 should we remove the banner from the lro files?